docs/development.md

```diff
@@ -0,0 +1,33 @@
+# Development
+
+Cattage can be developed in Tilt.
+
+1. Prepare a Linux box running Docker.
+2. Checkout this repository. 
+
+    ```console
+    $ git clone https://github.com/cybozu-go/cattage
+    ```
+
+3. Launch local Kubernetes cluster.
+
+    ```console
+    $ cd cattage
+    $ make dev
```

Comment:
It tried to use `controller-gen` and `kind` that existed in the PATH of the local environment, and could not start up the development environment well.
It would be nice if `make setup` in the project root downloads all necessary binaries such as kind in `e2e/Makefile` and controller-gen in `Makefile` in the project root to `cattage/bin` and uses them in `make dev` or `./bin/tilt up`.

Reply:
Please try aqua!